modules/snooker/src/main/ts/ephox/snooker/model/RunOperation.ts

```diff
@@ -136,7 +138,8 @@ const run = <RAW, INFO, GW extends GeneratorsWrapper>
     return output.bind((out) => {
       const newElements = Redraw.render(table, out.grid);
       const tableSizing = Optional.from(sizing).getOrThunk(() => TableSize.getTableSize(table));
-      adjustment(table, out.grid, tableSizing);
+      const resizing = Optional.from(resizeBehaviour).getOrThunk(ResizeBehaviour.preserveTable);
+      adjustment(table, out.grid, tableSizing, resizing, out.info);
```

Comment:
I thought you were going to change this to make it `table, grid, info, tableSizing, resizing`?

Reply:
We still have `Adjustments.adjustWidthTo` which has the same interface but only requires the first three, so I think it's probably easier to leave it?

Reply:
You're already doing a breaking change, so I think we should move it to where it makes more sense (ie group similar items).